raiden/console.py

```diff
@@ -228,8 +227,8 @@ def ping(self, peer):
             print("Error: peer {} not found in discovery".format(peer))
             return
 
-        nonce = self._ping_nonces[peer]
-        self._ping_nonces[peer] += 1
+        nonce = self._raiden.protocol.ping_nonces[peer]
+        self._raiden.protocol.ping_nonces[peer] += 1
         msg = Ping(nonce)
         self._raiden.sign(msg)
         return self._raiden.protocol.send_and_wait(peer.decode('hex'), msg)
```

Comment:
If `RaidenProtocol.send_ping` returns the `AsyncResult()` this method could wait on it, as in the `send_async`, then the nonce tracking would be kept only in the protocol layer. Also, I think it's a good idea to add a timeout argument.